skyportal/handlers/api/instrument.py

```diff
@@ -45,6 +45,15 @@ def post(self):
             telescope_id, self.current_user, raise_if_none=True, mode="read"
         )
 
+        sensitivity_data = data.get("sensitivity_data", None)
+        if sensitivity_data:
+            filters = data.get("filters", [])
+            for filter in sensitivity_data:
```

Comment:
Maybe instead of a loop we compare sets?

Reply:
good idea